src/popover.scss

```diff
@@ -219,6 +224,11 @@ $block: #{$fd-namespace}-popover;
       border-top-left-radius: 0;
       border-top-right-radius: 0;
 
+      .#{$block}__wrapper {
```

Comment:
If you introduce a new class for popover I would expect to have it documented on the popover page.